### DISCUSSION TITLE
Add and remove from Favorites

### DISCUSSION THREAD

**Discussion by paulearley** [punxaphil/custom-sonos-card#15] [Ideas]
Again thanks for a great component. Made  Sonos usable from Home Assistant. 
I already suggested this elsewhere but now the favorites function is usable with your card (it was too awkward in the Sonos app.), I wonder how difficult it would be to add these two buttons: "Add to Featured" and "Remove from Featured"?

The card is completely usable as is but you know how this goes, people always want more of a good thing!

Thanks again.

**Comment by punxaphil** [comment]
Yeah, would be awesome. Looks like it's not possible though. I can't see anything for it in neither the [Sonos integration](https://www.home-assistant.io/integrations/sonos/) or the [Media Player integration](https://www.home-assistant.io/integrations/media_player/).
